CMakeLists.txt

```diff
@@ -2,35 +2,45 @@ cmake_minimum_required(VERSION 2.8.3)
 project(moveit_core)
 
 if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
+  message("${PROJECT_NAME}: You did not request a specific build type: Choosing 'Release' for maximum performance")
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
 find_package(Boost REQUIRED system filesystem date_time thread iostreams)
+
+find_package(console_bridge REQUIRED)
+
+find_package(Eigen3 REQUIRED)
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(LIBFCL REQUIRED fcl)
+find_library(LIBFCL_LIBRARIES_FULL ${LIBFCL_LIBRARIES} ${LIBFCL_LIBRARY_DIRS})
+set(LIBFCL_LIBRARIES "${LIBFCL_LIBRARIES_FULL}")
+
+find_package(octomap REQUIRED)
+
+find_package(urdfdom REQUIRED)
+find_package(urdfdom_headers REQUIRED)
+
 find_package(catkin REQUIRED
 COMPONENTS
-  moveit_msgs
-  resource_retriever
```

Comment:
yes, you might want to look through the different commits for this request.
resource_retriever is not referenced in moveit_core.